service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/SupportedContentTypesTest.java

```diff
@@ -0,0 +1,26 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class SupportedContentTypesTest {
+
+    private SupportedContentTypes supportedContentTypes;
+
+    @BeforeEach
+    public void setUp() {
+        supportedContentTypes = new SupportedContentTypes();
+    }
+
+    @Test
+    public void When_CheckingIfNotSupportedContentTypeIsSupported_Expect_False() {
+        assertThat(supportedContentTypes.isContentTypeSupported("app/not-supported")).isFalse();
```

Comment:
Optional, I saw on comments of the ticket this functionality was to avoid executables running etc, could maybe use executable type here?